plugins/ecs-bridge/engine/interfaceip.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package engine
+
+import (
+	"github.com/aws/amazon-ecs-cni-plugins/pkg/netlinkwrapper"
+	"github.com/containernetworking/cni/pkg/ns"
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+)
+
+// getxInterfaceIPV4AddressContext wraps the parameters and the method to
+// get the ipv4 address of the container's veth's interface. This should
+// only be needed if there's a need to determine the IPv4 address of the
+// interface before executing the DEL command using the ipam plugin
+type getInterfaceIPV4AddressContext struct {
```

Comment:
I still have the `engine.GetInterfaceIPV4Address` method there in-case we need it in the future. You're right that we don't use this code today. We can delete it later so that we have it in the history if/when we need it in the future.